app/models/applicant.rb

```diff
@@ -1,6 +1,8 @@
 require 'date'
 
-class ClientDetail < ApplicationRecord
+class Applicant < ApplicationRecord
+
+  belongs_to :legal_aid_application
```

Comment:
Should it be this way around? Would it make more sense for an application to belong to an applicant? I talked to @mpw5 and I think we should keep it this way around as long as it's not going to be painful to change later

Reply:
Pretty sure a person can have multiple applications, but an application can only be for one person at a time. Something to ask the BAs